CHANGELOG.md

```diff
@@ -108,6 +108,10 @@ accordingly._
   [KS1019](https://github.com/KS1019/)
   [#4105](https://github.com/realm/SwiftLint/pull/4105)
 
+* Fix broken correction for `explicit_init` rule
+  [KS1019](https://github.com/KS1019/)
+  [#4109](https://github.com/realm/SwiftLint/pull/4109)
```

Comment:
Some minor nits here about changelog entry formatting. We usually have two trailing spaces to force a new line and only link to fixed issues, not PRs introducing the fix/change.